src/appmixer/google/blogger/README.md

```
@@ -0,0 +1,15 @@
+test@client.io
+
+TEST_SERVER_URL=http://localhost:2200 appmixer test auth login src/appmixer/google/auth.js -c 1038500334679-25o83vj5269qkj67frpgtgs0ndbaq2la.apps.googleusercontent.com -s CJrM6NIQ87qfIrhbYK3tq7Eh -o "https://www.googleapis.com/auth/blogger"
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)